### PR TITLE
Create ui-c8y-1019-22-1-disable-inventory-roles-page-for-external-users-if-inventory-5129-graft.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-57096](https://cumulocity.atlassian.net/browse/MTM-57096)
Original PR: [6157](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6157)

### CODE REVIEW

File: content/change-logs/platform-services/ui-c8y-1019-22-1-disable-inventory-roles-page-for-external-users-if-inventory-5129-graft.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: disable inventoryRoles page for external users if inventory… (#5129) [GRAFT][release/cd] (#6157)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1019-22-1-disable-inventory-roles-page-for-external-users-if-inventory-5129-graft.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Disable inventory roles page for external users if inventory is disabled
```

File: content/change-logs/platform-services/ui-c8y-1019-22-1-disable-inventory-roles-page-for-external-users-if-inventory-5129-graft.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57096
+version: 1019.22.1
+---
+disable inventoryRoles page for external users if inventory… (#5129) [GRAFT][release/cd] (#6157)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1019-22-1-disable-inventory-roles-page-for-external-users-if-inventory-5129-graft.md:17]: The Claude AI suggested to use the following body:
```suggestion
In an effort to improve the user experience and prevent confusion, the inventory roles page is now disabled for external users if the inventory feature is not enabled. Previously, external users were able to access the inventory roles page even if the inventory was disabled, which could lead to misunderstandings about the availability and functionality of the feature. With this change, external users will no longer see the inventory roles page when the inventory is turned off, providing a clearer and more consistent experience across the application.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by pink-mongoose — APPROVED** [review]

File: content/change-logs/platform-services/ui-c8y-1019-22-1-disable-inventory-roles-page-for-external-users-if-inventory-5129-graft.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57096
+version: 1019.22.1
+---
+In an effort to improve the user experience and prevent confusion, the inventory roles page is now disabled for external users if the inventory feature is not enabled. Previously, external users were able to access the inventory roles page even if the inventory was disabled, which could lead to misunderstandings about the availability and functionality of the feature. With this change, external users will no longer see the inventory roles page when the inventory is turned off, providing a clearer and more consistent experience across the application.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1019-22-1-disable-inventory-roles-page-for-external-users-if-inventory-5129-graft.md:17]:
```suggestion
In an effort to improve the user experience and prevent confusion, the **Inventory roles** tab is now disabled for external users if the inventory feature is not enabled. Previously, external users were able to access the **Inventory roles** tab even if the inventory was disabled, which could lead to misunderstandings about the availability and functionality of the feature. With this change, external users will no longer see the **Inventory roles** tab when the inventory is turned off, providing a clearer and more consistent experience across the application.
```

Comment (BeateRixen) [reply content/change-logs/platform-services/ui-c8y-1019-22-1-disable-inventory-roles-page-for-external-users-if-inventory-5129-graft.md:17]: What does "external user" mean here?
What does "inventory feature" mean? We should get that clearer. Is it a specific permission? What does it mean "inventory is disabled/turned off"? This is a bit fuzzy.

Comment (BeateRixen) [reply content/change-logs/platform-services/ui-c8y-1019-22-1-disable-inventory-roles-page-for-external-users-if-inventory-5129-graft.md:17]: @martin-sag Please follow up on this.

Comment (mmed-softwareag) [reply content/change-logs/platform-services/ui-c8y-1019-22-1-disable-inventory-roles-page-for-external-users-if-inventory-5129-graft.md]: @BeateRixen 
External user is the user who is managed by sso server

**Review by kalavathig1 — APPROVED** [review]

**Review by kalavathig1 — APPROVED** [review]

**Review by kalavathig1 — APPROVED** [review]

File: content/change-logs/platform-services/ui-c8y-1019-22-1-disable-inventory-roles-page-for-external-users-if-inventory-5129-graft.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57096
+version: 1019.22.1
+---
+Administrators can no longer change the global roles and inventory roles of a user managed by the SSO server if they are updated during each login. This option can be set in the access mapping of the SSO configuration.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1019-22-1-disable-inventory-roles-page-for-external-users-if-inventory-5129-graft.md:17]:
```suggestion
Administrators can no longer change the global roles and inventory roles of users managed by the single sign-on (SSO) server if they are updated during each login. This option can be set in the access mapping of the SSO configuration.
```

File: content/change-logs/platform-services/ui-c8y-1019-22-1-disable-inventory-roles-page-for-external-users-if-inventory-5129-graft.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57096
+version: 1019.22.1
+---
+Administrators can no longer change the global roles and inventory roles of users managed by the single sign-on (SSO) server if they are updated during each login. This option can be set in the access mapping of the SSO configuration.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1019-22-1-disable-inventory-roles-page-for-external-users-if-inventory-5129-graft.md:17]: This option can be set in the access mapping of the SSO configuration > @kalavathig1 What exactly can be set there?
if they are updated during each login > What does this mean? Do the roles have to be set again or or they taken from any other source?

Comment (kalavathig1) [reply content/change-logs/platform-services/ui-c8y-1019-22-1-disable-inventory-roles-page-for-external-users-if-inventory-5129-graft.md:17]:
```suggestion
Administrators can no longer change the global roles and inventory roles of users managed by the single sign-on (SSO) server if they are updated during each login. The option "
Roles selected in the rules below will be reassigned to a user on each log in and other ones will be cleared
" can be set in the access mapping of the SSO configuration
```

Comment (kalavathig1) [reply content/change-logs/platform-services/ui-c8y-1019-22-1-disable-inventory-roles-page-for-external-users-if-inventory-5129-graft.md:17]: @BeateRixen please check if it makes sense?

Comment (BeateRixen) [reply content/change-logs/platform-services/ui-c8y-1019-22-1-disable-inventory-roles-page-for-external-users-if-inventory-5129-graft.md:17]: Sounds much better. Please commit I I will do one more language review

**Review by mmed-softwareag — APPROVED** [review]

File: content/change-logs/platform-services/ui-c8y-1019-22-1-disable-inventory-roles-page-for-external-users-if-inventory-5129-graft.md [hunk 6]
```diff
@@ -12,6 +12,7 @@ build_artifact:
   - value: tc-pjJiURv9Y
     label: ui-c8y
 ticket: MTM-57096
-version:
+version: 1019.22.1
 ---
-Administrators can no longer change the global roles and inventory roles of a user managed by the SSO server if they are updated during each login. This option can be set in the access mapping of the SSO configuration.
+Administrators can no longer change the global roles and inventory roles of users managed by the single sign-on (SSO) server if they are updated during each login. The option "
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1019-22-1-disable-inventory-roles-page-for-external-users-if-inventory-5129-graft.md:17]:
```suggestion
Administrators can no longer change the global roles and inventory roles of users managed by the single sign-on (SSO) server if they are updated during login. 
```

File: content/change-logs/platform-services/ui-c8y-1019-22-1-disable-inventory-roles-page-for-external-users-if-inventory-5129-graft.md [hunk 7]
```diff
@@ -12,6 +12,7 @@ build_artifact:
...
+Roles selected in the rules below will be reassigned to a user on each log in and other ones will be cleared" can be set in the access mapping of the SSO configuration
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1019-22-1-disable-inventory-roles-page-for-external-users-if-inventory-5129-graft.md:18]:
```suggestion
The option "Roles selected in the rules below will be reassigned to a user on each log in and other ones will be cleared" can be set in the access mapping of the SSO configuration.
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**